packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "42.1.0",
+  "version": "42.1.1",
```

Comment:
Will a release be made after this is merged? If not, we should revert this and instead just update the web-components package version.

Reply:
Can do, not planning a release right away any more.